tests/test_learning_k_means.py

```diff
@@ -0,0 +1,65 @@
+"""
+Unit tests for landmarks space.
+"""
+
+import geomstats
+import geomstats.backend as gs
+import geomstats.tests
+
+from geomstats.geometry import hyperbolic_space
+from geomstats.geometry import hypersphere
+from geomstats.learning.k_means import KMeans
+
+
+class TestLearningKMeans(geomstats.tests.TestCase):
+    _multiprocess_can_split_ = True
+
+    def setUp(self):
+        pass
+
+    def test_hyperbolic_space_k_means(self):
+        x = gs.rand(100, 2)
+        manifold = hyperbolic_space.HyperbolicSpace(2)
+        metric = hyperbolic_space.HyperbolicMetric(2)
+        x = manifold.intrinsic_to_extrinsic_coords(x)
+        k_means = KMeans(2, metric, tol=1e-4)
+        # test learning
+        k_means.fit(x, max_iter=100)
```

Comment:
nice, it shows that it runs! Would it be possible to have a test with non-random data so that we can test the output of fit?